packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.types.ts

```diff
@@ -383,6 +383,11 @@ export interface IColumn {
    */
   onRender?: (item?: any, index?: number, column?: IColumn) => any;
 
+  /**
+   * If provider, can be used to render a custom column header divider
+   */
+  onRenderDivider?: (columnIndex: number, defaultRenderer: Function) => any;
```

Comment:
Our onRender functions typically use the IRenderFunction which provides the user with the original render function to be used in their on onRender. Is there a reason this wasn't used?

Reply:
Updated the definition.